mesh/nurbs.hpp

```diff
@@ -227,6 +230,20 @@ class NURBSExtension
    void CheckPatches();
    void CheckBdrPatches();
 
+   /** Checks the direction of the knotvectors in the patch based on
+       the patch orientation for patch @a p returns the direction of
+       the Knotvectors in @a kvdir.*/
+   void CheckKVDirection(int p, Array <int> &kvdir);
+   /**  Creates the extended array of KnotVectors */
+   void CreateExtendedKV();
+   /**  Updates the unique set of KnotVectors */
+   void UpdateUniqueKV();
+
+   /** Checks if the extended array of KnotVectors agrees with
+       the reduced set of KnotVectors. Returns false if it finds
+       a difference. */
+   bool InconsistentUniqueKVExtendedKV();
```

Comment:
Returning `true` in the consistent case suggests the name should be `ConsistentUniqueKVExtendedKV`.

Reply:
Good suggestion! Implemented.